### PR TITLE
Kb.Tabs QA

### PR DESCRIPTION
**Pull request by jakob223**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/common-adapters/tabs.tsx [hunk 1]
```diff
@@ -45,38 +47,64 @@ const TabText = ({selected, text}: {selected: boolean; text: string}) => (
...
-            <Kb.Divider style={selected ? styles.dividerSelected : styles.divider} />
-          </Kb.Box2>
-        </Kb.ClickableBox>
-      )
-    })}
-    {props.showProgressIndicator && <Kb.ProgressIndicator style={styles.progressIndicator} />}
-  </Kb.Box2>
-)
+const Tabs = <TitleT extends string>(props: Props<TitleT>) => {
+  const mobileTabMode = props.mobileTabModeOverride ?? (props.tabs.length > 3 ? 'scroll' : 'distribute')
```
Comment (buoyad) [inline shared/common-adapters/tabs.tsx:51]: :sheep: might clean up some of the checks below if this were undefined on non-mobile

File: shared/common-adapters/tabs.tsx [hunk 2]
```diff
@@ -88,30 +116,46 @@ const styles = Styles.styleSheetCreate(() => ({
...
+      height: 40,
+      width: 120,
+    },
+    isMobile: {
+      height: 48,
+    },
+  }),
   container: {
+    backgroundColor: Styles.globalColors.white,
     borderBottomColor: Styles.globalColors.black_10,
```
Comment (buoyad) [inline shared/common-adapters/tabs.tsx:131]: I think this border needs to be set on the scrollview as well, currently it ends at the edge of the screen. 
image

File: shared/common-adapters/tabs.tsx [hunk 3]
```diff
@@ -88,30 +116,46 @@ const styles = Styles.styleSheetCreate(() => ({
...
   icon: {
     alignSelf: 'center',
   },
   iconSelected: {
     alignSelf: 'center',
     color: Styles.globalColors.black,
   },
+  minWidthFull: {minWidth: '100%'},
+  mobileDistribute: Styles.platformStyles({isMobile: {flex: 1}}),
+  mobileScroll: Styles.platformStyles({isMobile: {width: 120}}),
```
Comment (buoyad) [inline shared/common-adapters/tabs.tsx:158]: does this need to be a fixed width?

Comment (jakob223) [reply shared/common-adapters/tabs.tsx:158]: No, but it is in the designs https://zpl.io/291XDQY

**Pull request merged**